plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/query/DynamicTableBuilder.java

```diff
@@ -42,8 +47,9 @@
     private static final CalciteSqlCompiler REQUEST_COMPILER = new CalciteSqlCompiler();
     private static final String COLUMN_KEY = "column";
     private static final String WILDCARD = "*";
-    public static final Set<String> DOUBLE_AGGREGATIONS = ImmutableSet.of("distinctcounthll", "avg");
```

Comment:
This was suggested by @findepi above as well - I can add it back, lmk.

Reply:
I'm ok either way, but I'm curious about *why* it was removed?

Reply:
@findepi suggested it's better to add as projection for approximate count which makes sense (I can do it in another pr). And for distinct count: if it's fully pushed to pinot it can crash a broker on a large table, but if the "select col1, col2... group by col1, col2" is pushed down and then trino does the "select col1, count(col2)..." it doesn't crash (tried this on large tables).

Reply:
Update: pushed down approx_distinct -> distinctcounthll in pinot.